apps/ubuntu_bootstrap/lib/pages/storage/passphrase/passphrase_page.dart

```diff
@@ -21,29 +21,48 @@ class PassphrasePage extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
+    final isTpm = ref.watch(
+      passphraseModelProvider.select((model) => model.isTpm),
+    );
     final lang = UbuntuBootstrapLocalizations.of(context);
+    final l10n = UbuntuLocalizations.of(context);
```

Comment:
Could we call this `ulang` for consistency with other pages?

Reply:
I've not seen `ulang` in other pages in the installer but I can change it if that's what you'd prefer?

Reply:
Right, seems like the installer localizations are rarely used in the same place as those coming from `ubuntu_localizations`, what I had in mind is actually part of the test utils: https://github.com/canonical/ubuntu-desktop-provision/blob/1cd4ada4883d3773e8bd0950f0f25ad2c28e1d19/packages/ubuntu_provision/test/test_utils.dart#L22-L30